packages/documentation/src/sidebar.js

```diff
@@ -9,23 +9,28 @@ module.exports = {
           name: 'Start up guide',
           items: [
             {
-              name: 'Set up VA.gov locally',
+              name: '1. Set up VA.gov locally',
```

Comment:
We may want to link these pages since they're written to be followed in order. These titles are verbs suggesting they instruct you on doing something. the other ones are nouns suggestion they're _about_ something. not sure if this is something we should stick to.

Reply:
Yeah, I think having some sort of "Next: blah blah" link at the bottom of each doc would work. That's what the external teams repo did for some stuff.

Reply:
I'll do that in a subsequent PR.